File: IT/Funzioni/Esercizi/Python/esercizio11.py
# file esercizio11.py
# -*- coding: utf-8 -*-

from __future__ import print_function

def multiple(a, b):
	if a == 0 or b == 0: return False
	return b % a == 0 if a <= b else a % b == 0

a = 5
b = 10
c = 11

print("{} è multiplo di {}? {}".format(b, a, multiple(a, b)))
print("{} è multiplo di {}? {}".format(c, a, multiple(c, a)))

File: IT/Funzioni/Esercizi/Java/Esercizio11.java
/* file Esercizio11.java */

public class Esercizio11 {
	static boolean multiple(int a, int b) {
		if (a == 0 || b == 0) return false;
		return a <= b ? b % a == 0 : a % b == 0;
	}

	public static void main(String [] args) {
		int a = 5;
		int b = 10;
		int c = 11;

		System.out.println(b + " è multiplo di " + a + "? " + multiple(a, b));
		System.out.println(c + " è multiplo di " + a + "? " + multiple(c, a));
	}
}

